src/sql/workbench/contrib/searchViewPane/browser/media/searchViewPane.css

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.searchExplorer-viewlet > .header {
```

Comment:
The class name here doesn't seem to make sense - this is supposed to be able to added to any viewlet right? Shouldn't this be something like `.search-view-pane > .header`?

(in general we should use - notation for css names too)